Controls/CkEditorMinimal/src/DotVVM.Contrib/Styles/DotVVM.Contrib.CkEditorMinimal.css

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
I don't think that we are going to need a CSS file for CKEditor, so I'd remove this file and corresponding resource.